install/migrations/update_9.5.x_to_10.0.0/cable.php

```diff
@@ -164,28 +173,32 @@
                             'netpoints_id'      => ['<>', 0]]);
 
       foreach ($datas as $id => $values) {
-
          //Load NetworkPort to get associated item
          $networkport = new NetworkPort();
          if ($networkport->getFromDB($values['networkports_id'])) {
             $sockets_id = $values['netpoints_id'];
             $socket = new Socket();
-               $socket->add([
-                  'id'              => $sockets_id,
-                  'position'        => $networkport->fields['logical_number'],
-                  'itemtype'        => $networkport->fields['itemtype'],
-                  'items_id'        => $networkport->fields['items_id'],
-                  'networkports_id' => $networkport->getID()
-               ]);
+            $socket->add([
+               'id'              => $sockets_id,
```

Comment:
You may have the same netpoint used on multiple networkport, but you cannot create multiple sockets having the same id.
There is no need to force the ID here, so you can just drop this line.
```suggestion
```

Reply:
rework done